dali/pipeline/operators/decoder/CMakeLists.txt

```diff
@@ -15,8 +15,10 @@
 # Get all the source files and dump test files
 if (BUILD_NVJPEG)
   list(APPEND DALI_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/nvjpeg_decoder.cu")
+  list(APPEND DALI_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/nvjpeg_decoder_new.cu")
```

Comment:
When you update CMake nvJPEG detection we can have if/else switch here. And then both nvJPEGDecoders should be named in the same way.
Also instead of `nvjpeg_decoder_new` we can try something more meaningful `nvjpeg_decoder_separated_stages`?

Reply:
Done